docs/audit-log-filter/audit-log-filter-tables.md

```diff
@@ -0,0 +1,109 @@
+# Audit Log Filter tables
+
+The Audit Log Filter plugin uses `mysql` system database tables in the `InnoDB` storage engine. These tables stores user account data and filter data. You can change the database used by the plugin with the `audit_log_filter_database` variable when you start the server.
```

Comment:
When you start the server, you can change the database used by the plugin with the `audit_log_filter_database` variable.

Reply:
Moved and rewritten